### PR TITLE
Only use spotbugs null checks

### PR DESCRIPTION
**Pull request by hendrikebbers**
This is a preparation of https://github.com/hashgraph/hedera-services/issues/4234 
fixes https://github.com/hashgraph/hedera-services/issues/4232
fixes https://github.com/hashgraph/hedera-services/issues/4334
Since we decided that we want to have our own well defined annotation lib in the future I refactored all modules to only depend on one (spotbugs) annotation lib. All others are removed from the classpath and con not be used.

### CODE REVIEW

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturns.java [hunk 1]
```diff
@@ -20,7 +20,15 @@
 import static com.hedera.services.state.expiry.removal.FungibleTreasuryReturns.UNFINISHED_NOOP_FUNGIBLE_RETURNS;
 import static com.hedera.services.state.expiry.removal.NonFungibleTreasuryReturns.FINISHED_NOOP_NON_FUNGIBLE_RETURNS;
 import static com.hedera.services.state.expiry.removal.NonFungibleTreasuryReturns.UNFINISHED_NOOP_NON_FUNGIBLE_RETURNS;
-import static com.hedera.services.throttling.MapAccessType.*;
+import static com.hedera.services.throttling.MapAccessType.ACCOUNTS_GET;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturns.java:23]: Did `spotlessApply` get back these ? Or is it Intellij setting ?

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/serdes/IoUtils.java [hunk 2]
```diff
@@ -77,10 +77,10 @@ public static <T> void writeNullable(
         }
     }
 
-    public static byte[] byteStream(JKeySerializer.StreamConsumer<DataOutputStream> consumer)
+    public static byte[] byteStream(final JKeySerializer.StreamConsumer<DataOutputStream> consumer)
             throws IOException {
-        try (ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
-            try (DataOutputStream dos = new DataOutputStream(bos)) {
+        try (final ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/serdes/IoUtils.java:82]: Adding `final`s wherever needed, is it IntelliJ setting ?

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! 💯

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/serdes/IoUtils.java (hunk 2 above)
Comment (hendrikebbers) [reply hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/serdes/IoUtils.java:82]: yes, I have activated it in IntelliJ

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturns.java (hunk 1 above)
Comment (hendrikebbers) [reply hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturns.java:23]: I have configured IntelliJ to not use * imports

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Pull request merged**